snapshot/policy/error_handling_policy.go

```diff
@@ -0,0 +1,37 @@
+package policy
+
+// ErrorHandlingPolicy controls error hadnling behavior when taking snapshots.
+type ErrorHandlingPolicy struct {
+	// IgnoreFileErrors controls whether or not snapshot operation should terminate when a file throws an error on being read
+	IgnoreFileErrors bool `json:"ignoreFileErrs,omitempty"`
+
+	// IgnoreFileErrorSet denotes whether the IgnoreFileErrors bool was actually set in this policy, or if it should be inherited
+	IgnoreFileErrorsSet bool `json:"ignoreFileErrsSet,omitempty"`
```

Comment:
would it be easier to use *bool here to avoid two flags (nil == inherit) ?

Reply:
great call; made the change.
https://github.com/kopia/kopia/pull/207/commits/87b8a11da489dcde0e4ba1ec93e9f4302b930e5b